packages/ckeditor5-link/src/linkediting.js

```diff
@@ -116,6 +117,9 @@ export default class LinkEditing extends Plugin {
 
 		// Handle typing over the link.
 		this._enableTypingOverLink();
+
+		// Handle removing the content after the link element.
+		this._enableRemovingLinkAttributesWhileRemovingEndOfLink();
```

Comment:
The comment doesn't match the method name. The method name is also long.

For me this method is \[enabling the\] handling what happens when user delete content after a link. So maybe

```
this._handleDeleteContentAfterLink();
```

Especially after reading method docs - it does that (with addition to handling selection attributes, etc). Also, the name might stay if we decide to handle also other delete content cases.